pulp_rpm/app/serializers/repository.py

```diff
@@ -69,6 +69,18 @@ class RpmRemoteSerializer(RemoteSerializer):
         allow_null=True,
     )
 
+    uln_user = serializers.CharField(
+        help_text=_("Username for ULN registration."),
+        required=False,
+        allow_null=True,
+    )
+
+    uln_pwd = serializers.CharField(
+        help_text=_("Password for ULN registration."),
+        required=False,
+        allow_null=True,
+    )
+
```

Comment:
I think we are missing the `server_url` (`uln_server_url` or `uln_server_base_url`) field in the serializer?

Reply:
Done ;)